app/components/views/GetStartedPage/OpenWallet/CreateWalletForm/ConfirmSeed/SeedEntry.js

```diff
@@ -16,20 +18,40 @@ class SeedEntry extends React.Component {
   constructor(props) {
     super(props);
     this.state = this.getInitialState();
+    this.getSeedWords = this.getSeedWords.bind(this);
   }
 
   getInitialState () {
     return {
-      value: Array()
+      currentHex: "",
+      currentWords: "",
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    if(nextProps.seedType != this.props.seedType) {
+      this.props.onChange(nextProps.seedType === "words"
+        ? this.state.currentWords : this.state.currentHex);
+    }
+  }
+
   render () {
     const { formatMessage } = this.props.intl;
-
+    const { seedType } = this.props;
+    if(seedType === "hex") {
+      return (
+        <Input
+          onChange={this.onChange}
+          value={this.state.currentHex}
+          onPaste={this.props.onPaste}
+          name='hexInput'
+          placeholder={formatMessage(messages.enterSeedPlaceholder)}
```

Comment:
Please make a new placeholder for the Hex Entry like: "Enter your seed in hex..." or something like that.

Reply:
absolutely.